examples/CommonInterfaces/CommonRigidBodyBase.h

```diff
@@ -7,7 +7,7 @@
 #include "CommonGUIHelperInterface.h"
 #include "CommonRenderInterface.h"
 #include "CommonCameraInterface.h"
-
+#include "BulletSoftBody/btSoftBody.h"
```

Comment:
Would it be possible to revert this change in CommonRigidBodyBase and move the btSoftBody.h into CommonDeformableBase.h?

Reply:
The SoftBody demo crashes now (due to changes in CommonRigidBodyBase.h), could you remove those changes (there should be no btSoftBody.h in that class)

Reply:
Done.